### DISCUSSION TITLE
New Crest display is very confusing

### DISCUSSION THREAD

**Discussion by Nokramx** [SavedInstances/SavedInstances#912] [General]
I don't get behind what the recent crest change wants to achieve. 
On one character it simply shows my current amount as the total amount, On the character right next to it it shows higher (but incorrect) total amounts. 
I'm completely confused what i'm even looking at.
This was perfect before the last update, please revert this.

![W52R5wY](https://github.com/user-attachments/assets/2d2a11ce-9291-48c3-a460-3b9790a8d744)

**Comment by LiangYuxuan** [comment]
Checkout #900 , the total here means the max value you can get (this week), so it subtracts the value you used which will not be available again.
